src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+
+    <session-factory>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/ticket_app?useSSL=false</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">12123312</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.hibernate.relations.model.Movie" />
+        <mapping class="mate.academy.hibernate.relations.model.Country" />
+        <mapping class="mate.academy.hibernate.relations.model.Actor" />
+
+    </session-factory>
+
+</hibernate-configuration>
```

Comment:
![image](https://github.com/user-attachments/assets/a8980e30-e46c-40b7-9af5-74dd49ba22bb)
always review your files and add empty line at the end where necessary